src/components/common/MarketSpan.tsx

```
@@ -0,0 +1,47 @@
+import React from 'react';
+import Image from 'next/image';
+import { findToken } from '@/utils/tokens';
+import { Market } from '@/utils/types';
+
+// Default chain ID (mainnet)
+const DEFAULT_CHAIN_ID = 1;
```

ESLint

[error] 'DEFAULT_CHAIN_ID' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)